src/tests/ftest/launch.py

```diff
@@ -1314,7 +1314,8 @@ def run_tests(test_files, tag_filter, args):
                     "{}/*.log*".format(test_log_dir),
                     args,
                     avocado_logs_dir,
-                    get_test_category(test_file["py"]))
+                    get_test_category(test_file["py"]),
+                    1800)
```

Comment:
```suggestion
                    get_test_category(test_file["py"]), 1800)
```

Reply:
That suggested change would go against https://pep8.org/#indentation